cachalot/tests/tests_decorators.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from functools import wraps
+from django.test.utils import override_settings
+
+logger = logging.getLogger(__name__)
+
+
+def all_final_sql_checks(func):
+    """
+    Runs test as two sub-tests:
+    one with CACHALOT_FINAL_SQL_CHECK setting True, one with False
+    """
+    @wraps(func)
+    def wrapper(self, *args, **kwargs):
+        for i, sql_check in enumerate((True, False)):
```

Comment:
@Fasther: what do you need the `i` and `enumerate` here for? It does not seem to be used.

Reply:
left-over :)